packages/api/src/address/address.controller.spec.ts

```diff
@@ -83,13 +88,15 @@ describe("AddressController", () => {
     });
 
     it("queries addresses by specified address", async () => {
-      await controller.getAddress(blockchainAddress);
+      const res = mock<Response>();
```

Comment:
Why redefining `res` instead of using `res` defined and initialized before each test?